xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -159,65 +162,65 @@ private void startLrsRpc() {
     if (!started) {
       return;
     }
-    checkState(lrsStream == null, "previous lbStream has not been cleared yet");
-    lrsStream = new LrsStream();
+    checkState(lrsStreamStatus == null, "previous lbStream has not been cleared yet");
     retryStopwatch.reset().start();
     Context prevContext = context.attach();
     try {
-      lrsStream.start();
+      lrsStreamStatus = new LrsStreamStatus();
     } finally {
       context.detach(prevContext);
     }
   }
 
-  private final class LrsStream {
+  private final class LrsStreamStatus {
```

Comment:
This also seems like the stream itself rather than the status of the stream.  Why not keep the original name?